WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -291,7 +291,7 @@ import Gridicons
         featuredImageView.image = nil
         var url = featuredImageURL
         featuredImageDesiredWidth = featuredImageView.frame.width
-        let size = CGSize(width:featuredImageDesiredWidth, height:100)
+        let size = CGSize(width:featuredImageDesiredWidth, height:featuredImageView.frame.height)
```

Comment:
It should be.  The frame is correct when the view awakes from its xib and since it no longer has a variable height, either for the iPad or when changing orientation we should be able to trust the frame.